nixos/modules/services/networking/v2raya.nix

```diff
@@ -0,0 +1,34 @@
+{ config, pkgs, lib, ... }:
+
+{
+  options = {
+    services.v2raya = {
+      enable = lib.options.mkEnableOption "the v2rayA service";
+    };
+  };
+
+  config = lib.mkIf config.services.v2raya.enable {
+    environment.systemPackages = [ pkgs.v2raya ];
```

Comment:
Is it necessary to add v2rayA into global environment? This module is about a service instead of a program.

Reply:
The service is directly related and depends on the program, so I think it is necessary to have it added to the global package list.